articles/agora-express-token-server/index.md

```diff
@@ -0,0 +1,364 @@
+In this tutorial, we'll be building a server using Node.js and Express to generate authentication tokens for the Agora SDKs.
+
+### Goals
+By the end of this tutorial, you’ll know:
+
+- How Agora's authentication works
+
+- How to generate a temporary token for testing
+
+- How to set up an Express server to dispatch authentication tokens for the Agora SDKs.
+
+### Agora
+Agora.io was founded in 2014. It's a service provider for real-time voice and video. Its core technology is real-time communication (RTC).
+Agora provides SDKs to build apps that require real-time engagement like:
+
+- Audio/Video Calls
+
+- Interactive Livestreaming (Audio/ Video)
+
+- Interactive Gaming
+
+- Real-Time Messaging (which is in BETA at the time of writing this article)
+
+Agora is a paid service, but the first 10,000 minutes are free every month. You can check their pricing [here](https://www.agora.io/en/pricing/).
+
+If you'd like to learn more about Agora, visit their [website](https://www.agora.io/en/) or read [this article](https://equalocean.com/analysis/201904121773).
+
+### Overview
+We'll be going through these steps in this article:
+
+1. Authentication in Agora
+2. Creating a project in Agora
+3. Generating a temporary token
+4. Setting up the server
+5. Generating authentication tokens
+6. Recap
+
+> If you want to take a look at the final code, check out the [GitHub Repo](https://github.com/zolomohan/agora-express-token-server).
+
+### Prerequisites
+This tutorial is for applications that use [Agora](https://www.agora.io/) and want to implement authentication for its users. If you are not using Agora in your application, then this tutorial is not for you.
+
+If you'd like to learn how to build a Livestream application with React Native and Agora, you can refer to [this tutorial](https://www.section.io/engineering-education/react-native-agora-livestreaming-app/).
+
+The fundamentals of Node.js and Express will not be covered in this tutorial. If you are not comfortable with the fundamentals, this is a [helpful tutorial](https://medium.com/@jaeger.rob/introduction-to-nodes-express-js-db5617047150) that you can go through before beginning with this project.
+
+### Authentication in Agora
+We need to provide authentication for the users when they join an RTC channel. There are three authentication mechanisms available in Agora.
```

Comment:
Consider having RTC in full on first use.
I'll give you an example below:

Full Name Of Something(FNOS)
After which you can use the abbreviation FNOS everywhere else.

Reply:
Done. I rewrote the line to not contain RTC.